### PR TITLE
Replace Broadway traffic cam with Dean Street

### PR DESCRIPTION
**Pull request by malssid**
- Updated seed.rb 
- Updated traffic cam test data

### CODE REVIEW

**Review by chewnoill — APPROVED** [review]

**Comment by chewnoill** [comment]
needs to reseed in prod after merge

File: db/seeds.rb [hunk 1]
```diff
@@ -140,9 +140,9 @@
 end.save!
 
 TrafficCam.find_or_initialize_by(
-  title: "Broadway"
```
Comment (chewnoill) [inline db/seeds.rb:143]: Do we need to remove this traffic cam from the database manually?

File: server-phoenix/lib/helios_web/test_data/traffic_cams.ex [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
...at\":\"2022-03-08T15:39:35.852Z\",\"updated_at\":\"2022-03-08T15:39:35.852Z\",\"feed_format\":\"image\"},{\"id\":3,\"title\":\"I-95 at Kinsley\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/95-22b%20I-95%20N%20@%20Kinsley%20Ave.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.854Z\",\"updated_at\":\"2022-03-08T15:39:35.854Z\",\"feed_format\":\"image\"},{\"id\":4,\"title\":\"Exit 10 (near I-295)\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/95-11%20I-95%20N%20@%20Toll%20Gate%20Rd.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.865Z\",\"updated_at\":\"2022-03-08T15:39:35.865Z\",\"feed_format\":\"image\"},{\"id\":5,\"title\":\"I-95 I-195 Split\",\"url\":\"http://www.dot.ri.gov/img/travel/camimages/195-1%20I-195%20W%20Split%20@%20I-95.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.867Z\",\"updated_at\":\"2022-03-08T15:39:35.867Z\",\"feed_format\":\"image\"},{\"id\":6,\"title\":\"Dean Street\",\"url\":\"https://www.dot.ri.gov/img/travel/camimages/6-1%20Rt%206%20E%20%20%2010%20N%20@%20Dean%20St.jpg\",\"location_id\":1,\"created_at\":\"2022-03-08T15:39:35.868Z\",\"updated_at\":\"2022-03-08T15:39:35.868Z\",\"feed_format\":\"image\"}]"
```
Comment (chewnoill) [inline server-phoenix/lib/helios_web/test_data/traffic_cams.ex:3]: Looks like we aren't storing these links in the database for the elixir version. Probably for the best, this seeding pattern seemed more complicated for our use case. Is there a way to make this blob of text easier for me to parse?  Ideally this diff would only be the things that changed.

File: db/seeds.rb (hunk 1 above)
Comment (cbarber) [reply db/seeds.rb:143]: This is more for consistency in development. We don't have a good way to execute this in production so I usually manually run it.

File: server-phoenix/lib/helios_web/test_data/traffic_cams.ex (hunk 2 above)
Comment (cbarber) [reply server-phoenix/lib/helios_web/test_data/traffic_cams.ex:3]: I think we can skip this entirely. This data is temporary.

**Pull request merged**